File: Assets/AltTester/AltDriver/Commands/InputActions/AltKeyUp.cs
namespace Altom.AltDriver.Commands
{
    public class AltKeyUp : AltBaseCommand
    {
        AltKeyUpParams cmdParams;

        public AltKeyUp(IDriverCommunication commHandler, AltKeyCode keyCode) : base(commHandler)
        {
            this.cmdParams = new AltKeyUpParams(keyCode);
        }
        public void Execute()
        {
            CommHandler.Send(cmdParams);
            var data = CommHandler.Recvall<string>(cmdParams);
            ValidateResponse("Ok", data);
        }
    }
}

File: Bindings~/java/src/main/java/ro/altom/alttester/Commands/InputActions/AltKeyUp.java
package ro.altom.alttester.Commands.InputActions;

import ro.altom.alttester.IMessageHandler;
import ro.altom.alttester.Commands.AltBaseCommand;

public class AltKeyUp extends AltBaseCommand {

    private AltKeyUpParams params;

    public AltKeyUp(IMessageHandler messageHandler, AltKeyUpParams params) {
        super(messageHandler);
        this.params = params;
    }

    public void Execute() {
        SendCommand(params);
        recvall(params, String.class);
    }
}
